test/add_packages.jl

```diff
@@ -0,0 +1,24 @@
+using Pkg
+
+# need this to use Conda
+# ENV["PYTHON"] = ""
+
+Pkg.add([
+    PackageSpec(url="https://github.com/tkf/PlotReferenceImages.jl", rev="pkg3"),
```

Comment:
TODO for me: Revert 17480637d721ccbeccc28cacaba7a0103cf5dfee once JuliaPlots/PlotReferenceImages.jl#30 is merged